salt/orchestrate/aws/map_templates/edx.yml

```diff
@@ -20,5 +20,23 @@ edx:
         roles:
           - edx
           - edx-{{ app_type }}
+
+  - edx-worker-{{ environment_name }}-{{ app_type }}-{{ id_num }}:
```

Comment:
This needs to be outside of the loop and scoped to the `edx_worker` namespace so that it will use the proper cloud profile.